stacker/util.py

```diff
@@ -545,6 +546,15 @@ def fetch_git_package(self, config):
         else:
             sys.path.append(cached_dir_path)
 
+        # If the configuration defines a set of remote config yamls to
+        # include, merge them into the main config
+        if config.get('configs'):
```

Comment:
I feel like any mucking with the actual config we do needs to happen inside the `config/` module, otherwise this could get pretty confusing fast.  Not sure the best way to do this, but for future maintainability/readability, it seems like the right design.